pilot/pkg/proxy/envoy/v1/testdata/gateway.yaml

```diff
@@ -0,0 +1,11 @@
+# TODO: implement workload selectors
+# selector:
+#   podRole: public-ingress
+servers:
```

Comment:
What happened to th rest of the header? I can’t see the gateway name.

Reply:
yeah these tests have a weird way of specifying the metadata and type: those are in code (See config_test.go), while the spec is in the file.  confusing, but we're just following the pattern here :-)